resources/views/admin/user/list.blade.php

```diff
@@ -0,0 +1,50 @@
+@extends('layouts.app')
+
+@section('title')
+<a href="#!" class="breadcrumb">@lang('admin.admin')</a>
+<a href="#!" class="breadcrumb">@lang('admin.user_management')</a>
+@endsection
+
+@section('content')
+
+<div class="row">
+    <div class="col s12">
+        <div class="card">
+            <div class="card-content">
+                <span class="card-title">@lang('admin.user_management')</span>
+                <table>
+                    <tbody>
+                        @foreach ($users as $user)
+                        <tr>
+                            <td>
+                                <b>{{ $user->name }}</b><br>
+                                {{ $user->email }}
+                            </td>
+                            <td>
+                                <div class="right">
+                                    @can('view', $user)
+                                    <a href="{{ route('admin.user.show', ['id' => $user->id]) }}" class="btn-floating waves-effect">
+                                        <i class="material-icons">remove_red_eye</i></a>
+                                    @endcan
+                                    @can('viewPermissionFor', $user)
+                                    <a href="{{ route('admin.permissions.show', ['id' => $user->id]) }}" class="btn-floating red waves-effect">
+                                        <i class="material-icons">lock</i></a>
```

Comment:
This icon's meaning is not clear here. Maybe we can use regular buttons, or delete this from here: we can access the permissions through the user page.

Reply:
Okay, I'll just remove.